src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -416,6 +418,35 @@ export const ArtworkAppFragmentContainer = createFragmentContainer(
         ...ArtworkAuctionCreateAlertHeader_artwork
       }
     `,
+  }
+)
+
+interface ArtworkResultProps {
+  artworkResult: ArtworkApp_artworkResult$data
+}
+const ArtworkResult = (props: ArtworkResultProps) => {
+  const { artworkResult } = props
+  const { __typename } = artworkResult
+
+  if (__typename === "Artwork") {
+    return <ArtworkAppFragmentContainer artwork={artworkResult} {...props} />
+  }
+
+  return <ArtworkErrorFragmentContainer artworkError={artworkResult} />
+}
+
+export const ArtworkResultFragmentContainer = createFragmentContainer(
```

Comment:
Just a heads up, no strong opinions - Unless we're doing a pagination or refetch, can use the `useFragment` hook

Reply:
Cool, I will try the hook form.

Reply:
Actually, there are a couple of fragments simultaneously in this fragment container, which might need to be refactored for `useFragment`, plus this file uses the former, so I'll punt and keep that as-is.

However, the stand-alone `ArtworkErrorApp` component(s) I'll give it a shot.

Reply:
Oh yea, in those other vanilla places it was so easy to use! Still need to define the 'fragment' as a stand-alone `graphql` string, but it's nice not having a fragment container component.